src/Lime.Transport.SignalR/EnvelopeHub.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System;
+using System.Collections.Concurrent;
+using System.Threading.Channels;
+using System.Threading.Tasks;
+using Lime.Protocol.Network;
+using Lime.Protocol.Serialization;
+using Microsoft.AspNetCore.Http.Connections;
+using Microsoft.AspNetCore.SignalR;
+
+namespace Lime.Transport.SignalR
+{
+    [System.Diagnostics.CodeAnalysis.SuppressMessage("Performance", "CA1812: Remove internal classes that are never instantiated", Justification = "The class is instantiated via reflection by ASP.NET")]
+    internal class EnvelopeHub : Hub
+    {
+        private readonly Channel<ITransport> _transportChannel;
+        private readonly ITraceWriter _traceWriter;
+        private readonly IEnvelopeSerializer _envelopeSerializer;
+        private readonly IHubContext<EnvelopeHub> _hubContext;
+        private readonly HubOptions _hubOptions;
+        private readonly HttpConnectionDispatcherOptions _httpConnectionDispatcherOptions;
+        private readonly EnvelopeHubOptions _envelopeHubOptions;
+        private readonly ConcurrentDictionary<string, Channel<string>> _clientChannels;
+
+        public EnvelopeHub(Channel<ITransport> transportChannel,
+                           ConcurrentDictionary<string, Channel<string>> clientChannels,
+                           IEnvelopeSerializer envelopeSerializer,
+                           IHubContext<EnvelopeHub> hubContext,
+                           HubOptions hubOptions,
+                           HttpConnectionDispatcherOptions httpConnectionDispatcherOptions,
+                           EnvelopeHubOptions envelopeHubOptions,
+                           ITraceWriter traceWriter = null)
```

Comment:
Indent arguments in a new line to keep the consistency with the codebase
```suggestion
        public EnvelopeHub(
            Channel<ITransport> transportChannel,
            ConcurrentDictionary<string, Channel<string>> clientChannels,
            IEnvelopeSerializer envelopeSerializer,
            IHubContext<EnvelopeHub> hubContext,
            HubOptions hubOptions,
            HttpConnectionDispatcherOptions httpConnectionDispatcherOptions,
            EnvelopeHubOptions envelopeHubOptions,
            ITraceWriter traceWriter = null)
```